xbmc/playlists/PlayListM3U.cpp

```diff
@@ -125,6 +129,21 @@ bool CPlayListM3U::Load(const std::string& strFileName)
         iEndOffset = atoi(strLine.substr(iComma).c_str());
       }
     }
+    else if (StringUtils::StartsWith(strLine, PropertyMarker)
+    || StringUtils::StartsWith(strLine, VLCOptMarker))
+    {
+      size_t iColon = strLine.find(":");
+      size_t iEqualSign = strLine.find("=");
+      if (iColon != std::string::npos &&
+        iEqualSign != std::string::npos &&
+        iEqualSign > iColon)
+      {
+        properties.push_back(std::make_pair(
+          StringUtils::Trim(strLine.substr(iColon, iEqualSign - iColon)),
```

Comment:
travis fails with:
PlayListM3U.cpp:142:72: error: no matching function for call to ‘StringUtils::Trim(std::basic_string<char>)’

           StringUtils::Trim(strLine.substr(iColon, iEqualSign - iColon)),

                                                                        ^

PlayListM3U.cpp:142:72: note: candidates are:

In file included from /home/travis/build/xbmc/xbmc/xbmc/music/Artist.h:28:0,

                 from /home/travis/build/xbmc/xbmc/xbmc/music/Album.h:29,

                 from /home/travis/build/xbmc/xbmc/xbmc/music/tags/MusicInfoTag.h:32,

                 from PlayListM3U.cpp:29:

/home/travis/build/xbmc/xbmc/xbmc/utils/StringUtils.h:73:23: note: static std::string& StringUtils::Trim(std::string&)

   static std::string& Trim(std::string &str);